dockerfiles/enclave.debug.Dockerfile

```diff
@@ -24,12 +25,19 @@ COPY go.mod .
 COPY go.sum .
 RUN go mod download
 
+FROM get-dependencies as build-enclave
```

Comment:
What happens if this image is built without the `enclave.Dockerfile` image ?
Will it know to build the  `enclave.Dockerfile` image so to have the `build-enclave` available ?

Reply:
It doesn't need the enclave.Dockerfile. There is no way to share stuff between dockerfiles (without building images first and having multiple steps)

Reply:
Yep, misread the dockerfiles. I was thinking of layer re-usage in the docker registry.